[2023-06-17T22:08:01.272+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: showpulse_dbt_dag.install_dbt manual__2023-06-17T22:08:00.010610+00:00 [queued]>
[2023-06-17T22:08:01.279+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: showpulse_dbt_dag.install_dbt manual__2023-06-17T22:08:00.010610+00:00 [queued]>
[2023-06-17T22:08:01.279+0000] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2023-06-17T22:08:01.280+0000] {taskinstance.py:1289} INFO - Starting attempt 1 of 1
[2023-06-17T22:08:01.280+0000] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2023-06-17T22:08:01.290+0000] {taskinstance.py:1309} INFO - Executing <Task(BashOperator): install_dbt> on 2023-06-17 22:08:00.010610+00:00
[2023-06-17T22:08:01.295+0000] {standard_task_runner.py:55} INFO - Started process 82 to run task
[2023-06-17T22:08:01.299+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'showpulse_dbt_dag', 'install_dbt', 'manual__2023-06-17T22:08:00.010610+00:00', '--job-id', '308', '--raw', '--subdir', 'DAGS_FOLDER/dbt_run.py', '--cfg-path', '/tmp/tmpre8xkqqx']
[2023-06-17T22:08:01.300+0000] {standard_task_runner.py:83} INFO - Job 308: Subtask install_dbt
[2023-06-17T22:08:01.353+0000] {task_command.py:389} INFO - Running <TaskInstance: showpulse_dbt_dag.install_dbt manual__2023-06-17T22:08:00.010610+00:00 [running]> on host 62cfed75eae1
[2023-06-17T22:08:01.403+0000] {taskinstance.py:1518} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=nburkett
AIRFLOW_CTX_DAG_ID=showpulse_dbt_dag
AIRFLOW_CTX_TASK_ID=install_dbt
AIRFLOW_CTX_EXECUTION_DATE=2023-06-17T22:08:00.010610+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-06-17T22:08:00.010610+00:00
[2023-06-17T22:08:01.404+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-06-17T22:08:01.405+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'pip3 install dbt-bigquery']
[2023-06-17T22:08:01.414+0000] {subprocess.py:86} INFO - Output:
[2023-06-17T22:08:01.689+0000] {subprocess.py:93} INFO - Defaulting to user installation because normal site-packages is not writeable
[2023-06-17T22:08:01.951+0000] {subprocess.py:93} INFO - Collecting dbt-bigquery
[2023-06-17T22:08:02.086+0000] {subprocess.py:93} INFO -   Downloading dbt_bigquery-1.5.2-py3-none-any.whl (55 kB)
[2023-06-17T22:08:02.110+0000] {subprocess.py:93} INFO -      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 55.8/55.8 kB 2.1 MB/s eta 0:00:00
[2023-06-17T22:08:02.211+0000] {subprocess.py:93} INFO - Collecting dbt-core~=1.5.0
[2023-06-17T22:08:02.231+0000] {subprocess.py:93} INFO -   Downloading dbt_core-1.5.1-py3-none-any.whl (947 kB)
[2023-06-17T22:08:02.312+0000] {subprocess.py:93} INFO -      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 947.9/947.9 kB 11.9 MB/s eta 0:00:00
[2023-06-17T22:08:02.418+0000] {subprocess.py:93} INFO - Collecting google-cloud-bigquery~=3.0
[2023-06-17T22:08:02.437+0000] {subprocess.py:93} INFO -   Downloading google_cloud_bigquery-3.11.1-py2.py3-none-any.whl (219 kB)
[2023-06-17T22:08:02.453+0000] {subprocess.py:93} INFO -      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 219.4/219.4 kB 14.9 MB/s eta 0:00:00
[2023-06-17T22:08:02.520+0000] {subprocess.py:93} INFO - Collecting agate~=1.6.3
[2023-06-17T22:08:02.544+0000] {subprocess.py:93} INFO -   Downloading agate-1.6.3-py2.py3-none-any.whl (100 kB)
[2023-06-17T22:08:02.548+0000] {subprocess.py:93} INFO -      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 100.5/100.5 kB 26.8 MB/s eta 0:00:00
[2023-06-17T22:08:02.550+0000] {subprocess.py:93} INFO - Requirement already satisfied: google-cloud-dataproc~=5.0 in /home/***/.local/lib/python3.7/site-packages (from dbt-bigquery) (5.0.0)
[2023-06-17T22:08:02.551+0000] {subprocess.py:93} INFO - Requirement already satisfied: google-cloud-storage~=2.4 in /home/***/.local/lib/python3.7/site-packages (from dbt-bigquery) (2.7.0)
[2023-06-17T22:08:02.561+0000] {subprocess.py:93} INFO - Requirement already satisfied: isodate>=0.5.4 in /home/***/.local/lib/python3.7/site-packages (from agate~=1.6.3->dbt-bigquery) (0.6.1)
[2023-06-17T22:08:02.584+0000] {subprocess.py:93} INFO - Collecting parsedatetime!=2.5,!=2.6,>=2.1
[2023-06-17T22:08:02.598+0000] {subprocess.py:93} INFO -   Downloading parsedatetime-2.4.tar.gz (58 kB)
[2023-06-17T22:08:02.607+0000] {subprocess.py:93} INFO -      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 58.8/58.8 kB 12.5 MB/s eta 0:00:00
[2023-06-17T22:08:02.617+0000] {subprocess.py:93} INFO -   Preparing metadata (setup.py): started
[2023-06-17T22:08:02.889+0000] {subprocess.py:93} INFO -   Preparing metadata (setup.py): finished with status 'done'
[2023-06-17T22:08:02.949+0000] {subprocess.py:93} INFO - Collecting pytimeparse>=1.1.5
[2023-06-17T22:08:02.974+0000] {subprocess.py:93} INFO -   Downloading pytimeparse-1.1.8-py2.py3-none-any.whl (10.0 kB)
[2023-06-17T22:08:02.978+0000] {subprocess.py:93} INFO - Requirement already satisfied: Babel>=2.0 in /home/***/.local/lib/python3.7/site-packages (from agate~=1.6.3->dbt-bigquery) (2.12.1)
[2023-06-17T22:08:02.978+0000] {subprocess.py:93} INFO - Requirement already satisfied: python-slugify>=1.2.1 in /home/***/.local/lib/python3.7/site-packages (from agate~=1.6.3->dbt-bigquery) (8.0.1)
[2023-06-17T22:08:02.978+0000] {subprocess.py:93} INFO - Requirement already satisfied: six>=1.9.0 in /home/***/.local/lib/python3.7/site-packages (from agate~=1.6.3->dbt-bigquery) (1.16.0)
[2023-06-17T22:08:03.060+0000] {subprocess.py:93} INFO - Collecting leather>=0.3.2
[2023-06-17T22:08:03.086+0000] {subprocess.py:93} INFO -   Downloading leather-0.3.4-py2.py3-none-any.whl (31 kB)
[2023-06-17T22:08:03.181+0000] {subprocess.py:93} INFO - Collecting dbt-extractor~=0.4.1
[2023-06-17T22:08:03.205+0000] {subprocess.py:93} INFO -   Downloading dbt_extractor-0.4.1-cp36-abi3-manylinux_2_17_aarch64.manylinux2014_aarch64.whl (1.1 MB)
[2023-06-17T22:08:03.265+0000] {subprocess.py:93} INFO -      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 1.1/1.1 MB 19.0 MB/s eta 0:00:00
[2023-06-17T22:08:03.267+0000] {subprocess.py:93} INFO - Requirement already satisfied: idna<4,>=2.5 in /home/***/.local/lib/python3.7/site-packages (from dbt-core~=1.5.0->dbt-bigquery) (3.4)
[2023-06-17T22:08:03.267+0000] {subprocess.py:93} INFO - Requirement already satisfied: pathspec<0.12,>=0.9 in /home/***/.local/lib/python3.7/site-packages (from dbt-core~=1.5.0->dbt-bigquery) (0.9.0)
[2023-06-17T22:08:03.350+0000] {subprocess.py:93} INFO - Collecting logbook<1.6,>=1.5
[2023-06-17T22:08:03.371+0000] {subprocess.py:93} INFO -   Downloading Logbook-1.5.3.tar.gz (85 kB)
[2023-06-17T22:08:03.376+0000] {subprocess.py:93} INFO -      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 85.8/85.8 kB 21.7 MB/s eta 0:00:00
[2023-06-17T22:08:03.387+0000] {subprocess.py:93} INFO -   Preparing metadata (setup.py): started
[2023-06-17T22:08:03.682+0000] {subprocess.py:93} INFO -   Preparing metadata (setup.py): finished with status 'done'
[2023-06-17T22:08:03.684+0000] {subprocess.py:93} INFO - Requirement already satisfied: werkzeug<3,>=1 in /home/***/.local/lib/python3.7/site-packages (from dbt-core~=1.5.0->dbt-bigquery) (2.2.3)
[2023-06-17T22:08:03.684+0000] {subprocess.py:93} INFO - Requirement already satisfied: sqlparse<0.4.4,>=0.2.3 in /home/***/.local/lib/python3.7/site-packages (from dbt-core~=1.5.0->dbt-bigquery) (0.4.3)
[2023-06-17T22:08:03.685+0000] {subprocess.py:93} INFO - Requirement already satisfied: pytz>=2015.7 in /home/***/.local/lib/python3.7/site-packages (from dbt-core~=1.5.0->dbt-bigquery) (2023.3)
[2023-06-17T22:08:03.686+0000] {subprocess.py:93} INFO - Requirement already satisfied: typing-extensions>=3.7.4 in /home/***/.local/lib/python3.7/site-packages (from dbt-core~=1.5.0->dbt-bigquery) (4.5.0)
[2023-06-17T22:08:03.686+0000] {subprocess.py:93} INFO - Requirement already satisfied: cffi<2.0.0,>=1.9 in /home/***/.local/lib/python3.7/site-packages (from dbt-core~=1.5.0->dbt-bigquery) (1.15.1)
[2023-06-17T22:08:03.687+0000] {subprocess.py:93} INFO - Requirement already satisfied: requests<3.0.0 in /home/***/.local/lib/python3.7/site-packages (from dbt-core~=1.5.0->dbt-bigquery) (2.28.2)
[2023-06-17T22:08:03.758+0000] {subprocess.py:93} INFO - Collecting mashumaro[msgpack]==3.6
[2023-06-17T22:08:03.779+0000] {subprocess.py:93} INFO -   Downloading mashumaro-3.6-py3-none-any.whl (66 kB)
[2023-06-17T22:08:03.783+0000] {subprocess.py:93} INFO -      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 66.5/66.5 kB 24.0 MB/s eta 0:00:00
[2023-06-17T22:08:03.785+0000] {subprocess.py:93} INFO - Requirement already satisfied: pyyaml>=6.0 in /home/***/.local/lib/python3.7/site-packages (from dbt-core~=1.5.0->dbt-bigquery) (6.0)
[2023-06-17T22:08:03.957+0000] {subprocess.py:93} INFO - Collecting protobuf>=4.0.0
[2023-06-17T22:08:03.977+0000] {subprocess.py:93} INFO -   Downloading protobuf-4.23.3-cp37-abi3-manylinux2014_aarch64.whl (303 kB)
[2023-06-17T22:08:03.992+0000] {subprocess.py:93} INFO -      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 303.5/303.5 kB 21.2 MB/s eta 0:00:00
[2023-06-17T22:08:03.994+0000] {subprocess.py:93} INFO - Requirement already satisfied: colorama<0.4.7,>=0.3.9 in /home/***/.local/lib/python3.7/site-packages (from dbt-core~=1.5.0->dbt-bigquery) (0.4.6)
[2023-06-17T22:08:03.994+0000] {subprocess.py:93} INFO - Requirement already satisfied: Jinja2==3.1.2 in /home/***/.local/lib/python3.7/site-packages (from dbt-core~=1.5.0->dbt-bigquery) (3.1.2)
[2023-06-17T22:08:03.995+0000] {subprocess.py:93} INFO - Requirement already satisfied: packaging>20.9 in /home/***/.local/lib/python3.7/site-packages (from dbt-core~=1.5.0->dbt-bigquery) (21.3)
[2023-06-17T22:08:04.062+0000] {subprocess.py:93} INFO - Collecting hologram<=0.0.16,>=0.0.14
[2023-06-17T22:08:04.088+0000] {subprocess.py:93} INFO -   Downloading hologram-0.0.16-py3-none-any.whl (11 kB)
[2023-06-17T22:08:04.137+0000] {subprocess.py:93} INFO - Collecting networkx<2.8.1,>=2.3
[2023-06-17T22:08:04.163+0000] {subprocess.py:93} INFO -   Downloading networkx-2.6.3-py3-none-any.whl (1.9 MB)
[2023-06-17T22:08:04.274+0000] {subprocess.py:93} INFO -      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 1.9/1.9 MB 17.4 MB/s eta 0:00:00
[2023-06-17T22:08:04.357+0000] {subprocess.py:93} INFO - Collecting minimal-snowplow-tracker==0.0.2
[2023-06-17T22:08:04.375+0000] {subprocess.py:93} INFO -   Downloading minimal-snowplow-tracker-0.0.2.tar.gz (12 kB)
[2023-06-17T22:08:04.382+0000] {subprocess.py:93} INFO -   Preparing metadata (setup.py): started
[2023-06-17T22:08:04.640+0000] {subprocess.py:93} INFO -   Preparing metadata (setup.py): finished with status 'done'
[2023-06-17T22:08:04.642+0000] {subprocess.py:93} INFO - Requirement already satisfied: click<9,>=7.0 in /home/***/.local/lib/python3.7/site-packages (from dbt-core~=1.5.0->dbt-bigquery) (8.1.3)
[2023-06-17T22:08:04.645+0000] {subprocess.py:93} INFO - Requirement already satisfied: MarkupSafe>=2.0 in /home/***/.local/lib/python3.7/site-packages (from Jinja2==3.1.2->dbt-core~=1.5.0->dbt-bigquery) (2.1.2)
[2023-06-17T22:08:04.651+0000] {subprocess.py:93} INFO - Requirement already satisfied: msgpack>=0.5.6 in /home/***/.local/lib/python3.7/site-packages (from mashumaro[msgpack]==3.6->dbt-core~=1.5.0->dbt-bigquery) (1.0.5)
[2023-06-17T22:08:04.723+0000] {subprocess.py:93} INFO - Requirement already satisfied: grpcio<2.0dev,>=1.47.0 in /home/***/.local/lib/python3.7/site-packages (from google-cloud-bigquery~=3.0->dbt-bigquery) (1.53.0)
[2023-06-17T22:08:04.724+0000] {subprocess.py:93} INFO - Requirement already satisfied: python-dateutil<3.0dev,>=2.7.2 in /home/***/.local/lib/python3.7/site-packages (from google-cloud-bigquery~=3.0->dbt-bigquery) (2.8.2)
[2023-06-17T22:08:04.725+0000] {subprocess.py:93} INFO - Requirement already satisfied: google-resumable-media<3.0dev,>=0.6.0 in /home/***/.local/lib/python3.7/site-packages (from google-cloud-bigquery~=3.0->dbt-bigquery) (2.4.1)
[2023-06-17T22:08:04.725+0000] {subprocess.py:93} INFO - Requirement already satisfied: proto-plus<2.0.0dev,>=1.15.0 in /home/***/.local/lib/python3.7/site-packages (from google-cloud-bigquery~=3.0->dbt-bigquery) (1.19.6)
[2023-06-17T22:08:04.726+0000] {subprocess.py:93} INFO - Requirement already satisfied: google-cloud-core<3.0.0dev,>=1.6.0 in /home/***/.local/lib/python3.7/site-packages (from google-cloud-bigquery~=3.0->dbt-bigquery) (2.3.2)
[2023-06-17T22:08:04.726+0000] {subprocess.py:93} INFO - Requirement already satisfied: google-api-core[grpc]!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.0,<3.0.0dev,>=1.31.5 in /home/***/.local/lib/python3.7/site-packages (from google-cloud-bigquery~=3.0->dbt-bigquery) (2.8.2)
[2023-06-17T22:08:04.829+0000] {subprocess.py:93} INFO - Collecting google-cloud-dataproc~=5.0
[2023-06-17T22:08:04.849+0000] {subprocess.py:93} INFO -   Downloading google_cloud_dataproc-5.4.1-py2.py3-none-any.whl (307 kB)
[2023-06-17T22:08:04.865+0000] {subprocess.py:93} INFO -      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 307.5/307.5 kB 21.5 MB/s eta 0:00:00
[2023-06-17T22:08:04.902+0000] {subprocess.py:93} INFO - Collecting proto-plus<2.0.0dev,>=1.15.0
[2023-06-17T22:08:04.933+0000] {subprocess.py:93} INFO -   Downloading proto_plus-1.22.2-py3-none-any.whl (47 kB)
[2023-06-17T22:08:04.937+0000] {subprocess.py:93} INFO -      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 47.9/47.9 kB 19.7 MB/s eta 0:00:00
[2023-06-17T22:08:04.938+0000] {subprocess.py:93} INFO - Requirement already satisfied: grpc-google-iam-v1<1.0.0dev,>=0.12.4 in /home/***/.local/lib/python3.7/site-packages (from google-cloud-dataproc~=5.0->dbt-bigquery) (0.12.4)
[2023-06-17T22:08:05.013+0000] {subprocess.py:93} INFO - Collecting google-api-core[grpc]!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.0,<3.0.0dev,>=1.31.5
[2023-06-17T22:08:05.038+0000] {subprocess.py:93} INFO -   Downloading google_api_core-2.11.1-py3-none-any.whl (120 kB)
[2023-06-17T22:08:05.044+0000] {subprocess.py:93} INFO -      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 120.5/120.5 kB 22.8 MB/s eta 0:00:00
[2023-06-17T22:08:05.050+0000] {subprocess.py:93} INFO - Requirement already satisfied: google-auth<3.0dev,>=1.25.0 in /home/***/.local/lib/python3.7/site-packages (from google-cloud-storage~=2.4->dbt-bigquery) (2.17.0)
[2023-06-17T22:08:05.054+0000] {subprocess.py:93} INFO - Requirement already satisfied: pycparser in /home/***/.local/lib/python3.7/site-packages (from cffi<2.0.0,>=1.9->dbt-core~=1.5.0->dbt-bigquery) (2.21)
[2023-06-17T22:08:05.056+0000] {subprocess.py:93} INFO - Requirement already satisfied: importlib-metadata in /home/***/.local/lib/python3.7/site-packages (from click<9,>=7.0->dbt-core~=1.5.0->dbt-bigquery) (4.13.0)
[2023-06-17T22:08:05.063+0000] {subprocess.py:93} INFO - Requirement already satisfied: googleapis-common-protos<2.0dev,>=1.56.2 in /home/***/.local/lib/python3.7/site-packages (from google-api-core[grpc]!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.0,<3.0.0dev,>=1.31.5->google-cloud-bigquery~=3.0->dbt-bigquery) (1.56.4)
[2023-06-17T22:08:05.095+0000] {subprocess.py:93} INFO - Requirement already satisfied: grpcio-status<2.0.dev0,>=1.33.2 in /home/***/.local/lib/python3.7/site-packages (from google-api-core[grpc]!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.0,<3.0.0dev,>=1.31.5->google-cloud-bigquery~=3.0->dbt-bigquery) (1.48.2)
[2023-06-17T22:08:05.108+0000] {subprocess.py:93} INFO - Requirement already satisfied: rsa<5,>=3.1.4 in /home/***/.local/lib/python3.7/site-packages (from google-auth<3.0dev,>=1.25.0->google-cloud-storage~=2.4->dbt-bigquery) (4.9)
[2023-06-17T22:08:05.109+0000] {subprocess.py:93} INFO - Requirement already satisfied: pyasn1-modules>=0.2.1 in /home/***/.local/lib/python3.7/site-packages (from google-auth<3.0dev,>=1.25.0->google-cloud-storage~=2.4->dbt-bigquery) (0.2.8)
[2023-06-17T22:08:05.110+0000] {subprocess.py:93} INFO - Requirement already satisfied: cachetools<6.0,>=2.0.0 in /home/***/.local/lib/python3.7/site-packages (from google-auth<3.0dev,>=1.25.0->google-cloud-storage~=2.4->dbt-bigquery) (5.3.0)
[2023-06-17T22:08:05.116+0000] {subprocess.py:93} INFO - Requirement already satisfied: google-crc32c<2.0dev,>=1.0 in /home/***/.local/lib/python3.7/site-packages (from google-resumable-media<3.0dev,>=0.6.0->google-cloud-bigquery~=3.0->dbt-bigquery) (1.5.0)
[2023-06-17T22:08:05.125+0000] {subprocess.py:93} INFO - Requirement already satisfied: jsonschema>=3.0 in /home/***/.local/lib/python3.7/site-packages (from hologram<=0.0.16,>=0.0.14->dbt-core~=1.5.0->dbt-bigquery) (4.17.3)
[2023-06-17T22:08:05.150+0000] {subprocess.py:93} INFO - Requirement already satisfied: pyparsing!=3.0.5,>=2.0.2 in /home/***/.local/lib/python3.7/site-packages (from packaging>20.9->dbt-core~=1.5.0->dbt-bigquery) (3.0.9)
[2023-06-17T22:08:05.153+0000] {subprocess.py:93} INFO - Requirement already satisfied: future in /home/***/.local/lib/python3.7/site-packages (from parsedatetime!=2.5,!=2.6,>=2.1->agate~=1.6.3->dbt-bigquery) (0.18.3)
[2023-06-17T22:08:05.192+0000] {subprocess.py:93} INFO - Requirement already satisfied: text-unidecode>=1.3 in /home/***/.local/lib/python3.7/site-packages (from python-slugify>=1.2.1->agate~=1.6.3->dbt-bigquery) (1.3)
[2023-06-17T22:08:05.205+0000] {subprocess.py:93} INFO - Requirement already satisfied: urllib3<1.27,>=1.21.1 in /home/***/.local/lib/python3.7/site-packages (from requests<3.0.0->dbt-core~=1.5.0->dbt-bigquery) (1.26.15)
[2023-06-17T22:08:05.206+0000] {subprocess.py:93} INFO - Requirement already satisfied: certifi>=2017.4.17 in /home/***/.local/lib/python3.7/site-packages (from requests<3.0.0->dbt-core~=1.5.0->dbt-bigquery) (2022.12.7)
[2023-06-17T22:08:05.206+0000] {subprocess.py:93} INFO - Requirement already satisfied: charset-normalizer<4,>=2 in /home/***/.local/lib/python3.7/site-packages (from requests<3.0.0->dbt-core~=1.5.0->dbt-bigquery) (2.1.1)
[2023-06-17T22:08:05.285+0000] {subprocess.py:93} INFO - Requirement already satisfied: importlib-resources>=1.4.0 in /home/***/.local/lib/python3.7/site-packages (from jsonschema>=3.0->hologram<=0.0.16,>=0.0.14->dbt-core~=1.5.0->dbt-bigquery) (5.12.0)
[2023-06-17T22:08:05.286+0000] {subprocess.py:93} INFO - Requirement already satisfied: pkgutil-resolve-name>=1.3.10 in /home/***/.local/lib/python3.7/site-packages (from jsonschema>=3.0->hologram<=0.0.16,>=0.0.14->dbt-core~=1.5.0->dbt-bigquery) (1.3.10)
[2023-06-17T22:08:05.286+0000] {subprocess.py:93} INFO - Requirement already satisfied: attrs>=17.4.0 in /home/***/.local/lib/python3.7/site-packages (from jsonschema>=3.0->hologram<=0.0.16,>=0.0.14->dbt-core~=1.5.0->dbt-bigquery) (22.2.0)
[2023-06-17T22:08:05.287+0000] {subprocess.py:93} INFO - Requirement already satisfied: pyrsistent!=0.17.0,!=0.17.1,!=0.17.2,>=0.14.0 in /home/***/.local/lib/python3.7/site-packages (from jsonschema>=3.0->hologram<=0.0.16,>=0.0.14->dbt-core~=1.5.0->dbt-bigquery) (0.19.3)
[2023-06-17T22:08:05.298+0000] {subprocess.py:93} INFO - Requirement already satisfied: pyasn1<0.5.0,>=0.4.6 in /home/***/.local/lib/python3.7/site-packages (from pyasn1-modules>=0.2.1->google-auth<3.0dev,>=1.25.0->google-cloud-storage~=2.4->dbt-bigquery) (0.4.8)
[2023-06-17T22:08:05.339+0000] {subprocess.py:93} INFO - Requirement already satisfied: zipp>=0.5 in /home/***/.local/lib/python3.7/site-packages (from importlib-metadata->click<9,>=7.0->dbt-core~=1.5.0->dbt-bigquery) (3.15.0)
[2023-06-17T22:08:05.405+0000] {subprocess.py:93} INFO - Building wheels for collected packages: minimal-snowplow-tracker, logbook, parsedatetime
[2023-06-17T22:08:05.406+0000] {subprocess.py:93} INFO -   Building wheel for minimal-snowplow-tracker (setup.py): started
[2023-06-17T22:08:05.790+0000] {subprocess.py:93} INFO -   Building wheel for minimal-snowplow-tracker (setup.py): finished with status 'done'
[2023-06-17T22:08:05.791+0000] {subprocess.py:93} INFO -   Created wheel for minimal-snowplow-tracker: filename=minimal_snowplow_tracker-0.0.2-py3-none-any.whl size=17800 sha256=46b469e8ffef60f8d4c77f2aece446b02b2aab8d168b6fb53ff3596c04938c97
[2023-06-17T22:08:05.791+0000] {subprocess.py:93} INFO -   Stored in directory: /home/***/.cache/pip/wheels/9d/ee/55/8faefee41ca3bccb25152e0159002567575047be7ef1ce8ac1
[2023-06-17T22:08:05.793+0000] {subprocess.py:93} INFO -   Building wheel for logbook (setup.py): started
[2023-06-17T22:08:06.235+0000] {subprocess.py:93} INFO -   Building wheel for logbook (setup.py): finished with status 'done'
[2023-06-17T22:08:06.236+0000] {subprocess.py:93} INFO -   Created wheel for logbook: filename=Logbook-1.5.3-cp37-cp37m-linux_aarch64.whl size=65613 sha256=e6126da237599589e64c9842a74369968d320d2cf94ad5e4896fcc6f48e32fa9
[2023-06-17T22:08:06.236+0000] {subprocess.py:93} INFO -   Stored in directory: /home/***/.cache/pip/wheels/95/4b/a3/fc9b9d17a06a956767a8dc74058ba64d4306653df2289094cc
[2023-06-17T22:08:06.238+0000] {subprocess.py:93} INFO -   Building wheel for parsedatetime (setup.py): started
[2023-06-17T22:08:06.616+0000] {subprocess.py:93} INFO -   Building wheel for parsedatetime (setup.py): finished with status 'done'
[2023-06-17T22:08:06.617+0000] {subprocess.py:93} INFO -   Created wheel for parsedatetime: filename=parsedatetime-2.4-py3-none-any.whl size=42773 sha256=5e218c79eb76a7a33d8590612509b030e4d67baee9e36f25adc263142bb870e5
[2023-06-17T22:08:06.617+0000] {subprocess.py:93} INFO -   Stored in directory: /home/***/.cache/pip/wheels/68/0f/b7/539a31a4068565a53913d7f8cfc6c7521cca2ea091b8a72048
[2023-06-17T22:08:06.619+0000] {subprocess.py:93} INFO - Successfully built minimal-snowplow-tracker logbook parsedatetime
[2023-06-17T22:08:08.905+0000] {subprocess.py:93} INFO - Installing collected packages: pytimeparse, logbook, protobuf, parsedatetime, networkx, mashumaro, leather, dbt-extractor, proto-plus, minimal-snowplow-tracker, agate, hologram, google-api-core, dbt-core, google-cloud-dataproc, google-cloud-bigquery, dbt-bigquery
[2023-06-17T22:08:08.935+0000] {subprocess.py:93} INFO -   Attempting uninstall: protobuf
[2023-06-17T22:08:08.936+0000] {subprocess.py:93} INFO -     Found existing installation: protobuf 3.20.0
[2023-06-17T22:08:08.946+0000] {subprocess.py:93} INFO -     Uninstalling protobuf-3.20.0:
[2023-06-17T22:08:09.222+0000] {subprocess.py:93} INFO -       Successfully uninstalled protobuf-3.20.0
[2023-06-17T22:08:09.878+0000] {subprocess.py:93} INFO -   Attempting uninstall: proto-plus
[2023-06-17T22:08:09.879+0000] {subprocess.py:93} INFO -     Found existing installation: proto-plus 1.19.6
[2023-06-17T22:08:09.884+0000] {subprocess.py:93} INFO -     Uninstalling proto-plus-1.19.6:
[2023-06-17T22:08:09.889+0000] {subprocess.py:93} INFO -       Successfully uninstalled proto-plus-1.19.6
[2023-06-17T22:08:09.956+0000] {subprocess.py:93} INFO -   Attempting uninstall: google-api-core
[2023-06-17T22:08:09.956+0000] {subprocess.py:93} INFO -     Found existing installation: google-api-core 2.8.2
[2023-06-17T22:08:09.966+0000] {subprocess.py:93} INFO -     Uninstalling google-api-core-2.8.2:
[2023-06-17T22:08:10.131+0000] {subprocess.py:93} INFO -       Successfully uninstalled google-api-core-2.8.2
[2023-06-17T22:08:10.343+0000] {subprocess.py:93} INFO -   Attempting uninstall: google-cloud-dataproc
[2023-06-17T22:08:10.344+0000] {subprocess.py:93} INFO -     Found existing installation: google-cloud-dataproc 5.0.0
[2023-06-17T22:08:10.357+0000] {subprocess.py:93} INFO -     Uninstalling google-cloud-dataproc-5.0.0:
[2023-06-17T22:08:10.529+0000] {subprocess.py:93} INFO -       Successfully uninstalled google-cloud-dataproc-5.0.0
[2023-06-17T22:08:10.616+0000] {subprocess.py:93} INFO -   Attempting uninstall: google-cloud-bigquery
[2023-06-17T22:08:10.617+0000] {subprocess.py:93} INFO -     Found existing installation: google-cloud-bigquery 2.34.4
[2023-06-17T22:08:10.628+0000] {subprocess.py:93} INFO -     Uninstalling google-cloud-bigquery-2.34.4:
[2023-06-17T22:08:10.796+0000] {subprocess.py:93} INFO -       Successfully uninstalled google-cloud-bigquery-2.34.4
[2023-06-17T22:08:10.912+0000] {subprocess.py:93} INFO - ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
[2023-06-17T22:08:10.913+0000] {subprocess.py:93} INFO - mysql-connector-python 8.0.32 requires protobuf<=3.20.3,>=3.11.0, but you have protobuf 4.23.3 which is incompatible.
[2023-06-17T22:08:10.914+0000] {subprocess.py:93} INFO - google-cloud-workflows 1.7.1 requires protobuf<4.0.0dev,>=3.19.0, but you have protobuf 4.23.3 which is incompatible.
[2023-06-17T22:08:10.914+0000] {subprocess.py:93} INFO - google-cloud-vision 1.0.2 requires protobuf<4.0.0dev, but you have protobuf 4.23.3 which is incompatible.
[2023-06-17T22:08:10.915+0000] {subprocess.py:93} INFO - google-cloud-videointelligence 1.16.3 requires protobuf<4.0.0dev, but you have protobuf 4.23.3 which is incompatible.
[2023-06-17T22:08:10.915+0000] {subprocess.py:93} INFO - google-cloud-translate 1.7.2 requires protobuf<4.0.0dev, but you have protobuf 4.23.3 which is incompatible.
[2023-06-17T22:08:10.916+0000] {subprocess.py:93} INFO - google-cloud-texttospeech 1.0.3 requires protobuf<4.0.0dev, but you have protobuf 4.23.3 which is incompatible.
[2023-06-17T22:08:10.916+0000] {subprocess.py:93} INFO - google-cloud-tasks 2.10.1 requires protobuf<4.0.0dev,>=3.19.0, but you have protobuf 4.23.3 which is incompatible.
[2023-06-17T22:08:10.917+0000] {subprocess.py:93} INFO - google-cloud-speech 1.3.4 requires protobuf<4.0.0dev, but you have protobuf 4.23.3 which is incompatible.
[2023-06-17T22:08:10.918+0000] {subprocess.py:93} INFO - google-cloud-spanner 1.19.3 requires protobuf<4.0.0dev, but you have protobuf 4.23.3 which is incompatible.
[2023-06-17T22:08:10.919+0000] {subprocess.py:93} INFO - google-cloud-secret-manager 1.0.2 requires protobuf<4.0.0dev, but you have protobuf 4.23.3 which is incompatible.
[2023-06-17T22:08:10.920+0000] {subprocess.py:93} INFO - google-cloud-resource-manager 1.6.0 requires protobuf<4.0.0dev,>=3.19.0, but you have protobuf 4.23.3 which is incompatible.
[2023-06-17T22:08:10.920+0000] {subprocess.py:93} INFO - google-cloud-redis 2.9.0 requires protobuf<4.0.0dev,>=3.19.0, but you have protobuf 4.23.3 which is incompatible.
[2023-06-17T22:08:10.921+0000] {subprocess.py:93} INFO - google-cloud-pubsub 2.13.5 requires protobuf<4.0.0dev,>=3.19.0, but you have protobuf 4.23.3 which is incompatible.
[2023-06-17T22:08:10.921+0000] {subprocess.py:93} INFO - google-cloud-os-login 2.7.1 requires protobuf<4.0.0dev,>=3.19.0, but you have protobuf 4.23.3 which is incompatible.
[2023-06-17T22:08:10.922+0000] {subprocess.py:93} INFO - google-cloud-orchestration-*** 1.4.1 requires protobuf<4.0.0dev,>=3.19.0, but you have protobuf 4.23.3 which is incompatible.
[2023-06-17T22:08:10.922+0000] {subprocess.py:93} INFO - google-cloud-monitoring 2.11.0 requires protobuf<4.0.0dev,>=3.19.0, but you have protobuf 4.23.3 which is incompatible.
[2023-06-17T22:08:10.923+0000] {subprocess.py:93} INFO - google-cloud-memcache 1.4.1 requires protobuf<4.0.0dev,>=3.19.0, but you have protobuf 4.23.3 which is incompatible.
[2023-06-17T22:08:10.923+0000] {subprocess.py:93} INFO - google-cloud-logging 3.2.1 requires protobuf<4.0.0dev,>=3.19.0, but you have protobuf 4.23.3 which is incompatible.
[2023-06-17T22:08:10.924+0000] {subprocess.py:93} INFO - google-cloud-language 1.3.2 requires protobuf<4.0.0dev, but you have protobuf 4.23.3 which is incompatible.
[2023-06-17T22:08:10.924+0000] {subprocess.py:93} INFO - google-cloud-kms 2.12.0 requires protobuf<4.0.0dev,>=3.19.0, but you have protobuf 4.23.3 which is incompatible.
[2023-06-17T22:08:10.925+0000] {subprocess.py:93} INFO - google-cloud-dlp 3.7.1 requires protobuf<4.0.0dev,>=3.19.0, but you have protobuf 4.23.3 which is incompatible.
[2023-06-17T22:08:10.925+0000] {subprocess.py:93} INFO - google-cloud-dataproc-metastore 1.6.0 requires protobuf<4.0.0dev,>=3.19.0, but you have protobuf 4.23.3 which is incompatible.
[2023-06-17T22:08:10.926+0000] {subprocess.py:93} INFO - google-cloud-dataplex 1.1.0 requires protobuf<4.0.0dev,>=3.19.0, but you have protobuf 4.23.3 which is incompatible.
[2023-06-17T22:08:10.926+0000] {subprocess.py:93} INFO - google-cloud-dataform 0.2.0 requires protobuf<4.0.0dev,>=3.19.0, but you have protobuf 4.23.3 which is incompatible.
[2023-06-17T22:08:10.927+0000] {subprocess.py:93} INFO - google-cloud-dataflow-client 0.5.4 requires protobuf<4.0.0dev,>=3.19.0, but you have protobuf 4.23.3 which is incompatible.
[2023-06-17T22:08:10.927+0000] {subprocess.py:93} INFO - google-cloud-datacatalog 3.9.0 requires protobuf<4.0.0dev,>=3.19.0, but you have protobuf 4.23.3 which is incompatible.
[2023-06-17T22:08:10.928+0000] {subprocess.py:93} INFO - google-cloud-container 2.11.1 requires protobuf<4.0.0dev,>=3.19.0, but you have protobuf 4.23.3 which is incompatible.
[2023-06-17T22:08:10.928+0000] {subprocess.py:93} INFO - google-cloud-build 3.9.0 requires protobuf<4.0.0dev,>=3.19.0, but you have protobuf 4.23.3 which is incompatible.
[2023-06-17T22:08:10.929+0000] {subprocess.py:93} INFO - google-cloud-bigtable 2.11.1 requires protobuf<4.0.0dev,>=3.19.0, but you have protobuf 4.23.3 which is incompatible.
[2023-06-17T22:08:10.929+0000] {subprocess.py:93} INFO - google-cloud-bigquery-storage 2.14.1 requires protobuf<4.0.0dev,>=3.19.0, but you have protobuf 4.23.3 which is incompatible.
[2023-06-17T22:08:10.929+0000] {subprocess.py:93} INFO - google-cloud-bigquery-datatransfer 3.7.0 requires protobuf<4.0.0dev,>=3.19.0, but you have protobuf 4.23.3 which is incompatible.
[2023-06-17T22:08:10.930+0000] {subprocess.py:93} INFO - google-cloud-automl 2.8.0 requires protobuf<4.0.0dev,>=3.19.0, but you have protobuf 4.23.3 which is incompatible.
[2023-06-17T22:08:10.930+0000] {subprocess.py:93} INFO - google-cloud-appengine-logging 1.1.3 requires protobuf<4.0.0dev,>=3.19.0, but you have protobuf 4.23.3 which is incompatible.
[2023-06-17T22:08:10.931+0000] {subprocess.py:93} INFO - google-cloud-aiplatform 1.16.1 requires google-cloud-bigquery<3.0.0dev,>=1.15.0, but you have google-cloud-bigquery 3.11.1 which is incompatible.
[2023-06-17T22:08:10.931+0000] {subprocess.py:93} INFO - google-cloud-aiplatform 1.16.1 requires protobuf<4.0.0dev,>=3.19.0, but you have protobuf 4.23.3 which is incompatible.
[2023-06-17T22:08:10.932+0000] {subprocess.py:93} INFO - google-ads 18.0.0 requires proto-plus==1.19.6, but you have proto-plus 1.22.2 which is incompatible.
[2023-06-17T22:08:10.932+0000] {subprocess.py:93} INFO - google-ads 18.0.0 requires protobuf!=3.18.*,!=3.19.*,<=3.20.0,>=3.12.0, but you have protobuf 4.23.3 which is incompatible.
[2023-06-17T22:08:10.932+0000] {subprocess.py:93} INFO - Successfully installed agate-1.6.3 dbt-bigquery-1.5.2 dbt-core-1.5.1 dbt-extractor-0.4.1 google-api-core-2.11.1 google-cloud-bigquery-3.11.1 google-cloud-dataproc-5.4.1 hologram-0.0.16 leather-0.3.4 logbook-1.5.3 mashumaro-3.6 minimal-snowplow-tracker-0.0.2 networkx-2.6.3 parsedatetime-2.4 proto-plus-1.22.2 protobuf-4.23.3 pytimeparse-1.1.8
[2023-06-17T22:08:11.009+0000] {subprocess.py:93} INFO - 
[2023-06-17T22:08:11.010+0000] {subprocess.py:93} INFO - [notice] A new release of pip is available: 23.0.1 -> 23.1.2
[2023-06-17T22:08:11.011+0000] {subprocess.py:93} INFO - [notice] To update, run: python -m pip install --upgrade pip
[2023-06-17T22:08:11.129+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-06-17T22:08:11.152+0000] {taskinstance.py:1332} INFO - Marking task as SUCCESS. dag_id=showpulse_dbt_dag, task_id=install_dbt, execution_date=20230617T220800, start_date=20230617T220801, end_date=20230617T220811
[2023-06-17T22:08:11.183+0000] {local_task_job.py:212} INFO - Task exited with return code 0
[2023-06-17T22:08:11.202+0000] {taskinstance.py:2596} INFO - 1 downstream tasks scheduled from follow-on schedule check
